test/prototype_transforms_kernel_infos.py

```diff
@@ -62,7 +67,7 @@ def wrapper(image_tensor, *other_args, **kwargs):
 
 
 def sample_inputs_horizontal_flip_image_tensor():
-    for image_loader in make_image_loaders(dtypes=[torch.float32]):
+    for image_loader in make_image_loaders(sizes=[None], dtypes=[torch.float32]):
```

Comment:
This is an effort to keep the total number of tests moderate. Since the sample inputs should only cover all branches, we don't need to test multiple image sizes here. 

There are multiple places where I constrained the number of sample inputs below. I've checked the kernels and made sure that we cover all branches, but nothing more. The extensive tests should only happen in the reference inputs.